### ISSUE TITLE
How to finetune wan with dpo?

### ISSUE THREAD

**Issue by JeffreyYzh** [yifan123/flow_grpo#54]
Hello, I am currently trying to use the flow DPO algorithm from the VideoReward paper to fine-tune Wan2.1. Specifically, my experimental setup includes rank 256, alpha 256, a learning rate of 5e-6, and a batch size of 48. In addition, I set beta to 2000 during DPO training. My dataset contains 8000 DPO pairs(with wan generated pictures as lose sample and real pictures as win samples), and I trained for one epoch. After training, I performed inference using LoRA. However, I found that the generated videos differ very slightly from the original ones (reflected in local areas of objects), and I’m concerned there might be some issues with the training parameters or the code. I’d like to seek your advice—below is my TensorBoard curve. Thank you so much!

<img width="1747" height="862" alt="Image" src="https://github.com/user-attachments/assets/f08d7f6e-a0a6-4841-b7e2-4078a8caf7d7" />

https://github.com/user-attachments/assets/9722bc89-d116-486c-ba99-260ee0c2cbd3
https://github.com/user-attachments/assets/967da950-9df3-47e1-8ba0-d19b4e2731b1

**Comment by yifan123** [comment]
Can you show the implicit_acc

**Comment by JeffreyYzh** [comment]
Thanks for your advice! I will try it later. 
In addition, I noticed some abnormal loss values. Does this mean that some pairs have differences that are too large, or is there another reason? For the other recorded variables, such as w_diff, l_diff, and w_l_diff, what is their normal trend of variation? Are there any other issues with these training curves?
Thank you so much for your time!
